src/interfaces/user.type.ts

```diff
@@ -1,8 +1,28 @@
 type userType = {
-  github_display_name: String,
-  github_id: string,
   id: string,
+  username?: string,
+  github_id: string,
+  github_display_name: String | null,
```

Comment:
Maybe a typographical mistake
string is the TypeScript string type, which you can use to type variables, parameters and return values.

Reply:
yes, will make the change